togeojson.js

```diff
@@ -238,6 +239,9 @@ var toGeoJSON = (function() {
                         properties[simpleDatas[i].getAttribute('name')] = nodeVal(simpleDatas[i]);
                     }
                 }
+                if (typeof visibility !== undefined && visibility != null) {
```

Comment:
the typeof is not necessary here, you can simply ask `if (visibility)`.

typeof tests are necessary if there's a chance the variable has not been declared, and asking its value would trigger a ReferenceError. In this code, you can be confident that the variable is defined.